models/Announcement.ts

```diff
@@ -0,0 +1,10 @@
+import mongoose from "mongoose";
+
+const AnnouncementSchema = new mongoose.Schema({
+	title: String,
```

Comment:
I was under the impression that there would also be a date string in the announcement, unless I am mistaken

Reply:
It’s not stored in the database under an explicit field, it’s implicitly stored in the ObjectId.
The date is extracted AFTER fetching from the DB, server side (see get_announcements.ts)